zuul.d/kuttl.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- job:
+    name: cifmw-base-kuttl
+    nodeset: centos-9-crc-xxl
+    timeout: 5400
+    abstract: true
+    vars:
+      crc_parameters: "--memory 20000 --disk-size 120 --cpus 6"
+    parent: base-crc
```

Comment:
I suppose this parent job takes care of setting up the crc cluster, right?

Reply:
yes, correct!